chef_master/source/ohai.rst

```diff
@@ -218,6 +218,8 @@ For attributes that contain slashes (``/``) within the attribute value, such as
 
 Default Plugins
 =====================================================
+.. warning:: The system_profile plugin will be removed from Chef/Ohai 15 in April 2019. This plugin does not correctly return data on modern Mac systems. Additionally the same data is provided by the hardware plugin, which has a format that is simpler to consume. Removing this plugin will reduce Ohai return by ~3 seconds and greatly reduce the size of the node object on the Chef server.
```

Comment:
We usually just create a deprecation page on the deprecations page for these. There's too many to track with warnings.

Reply:
@tas50 Is there a certain code that needs to be used for this on the depreciations page?